tests/unit/activities.unitTest.ts

```diff
@@ -315,26 +294,116 @@ describe("endActivity", () => {
 
             // If the call does not throw errors, it is successful
             return activityService.endActivity(activityId)
-            .catch((error: HTTPResponse) => {
-                // The creation should succeed
-                console.log(error);
-                expect.fail();
-            });
+                .catch((error: HTTPResponse) => {
+                    expect.fail();
+                });
         });
     });
 
     context("when the activity has already ended", () => {
         it("should return an error", async () => {
             return activityService.endActivity(activityId)
-            .then(() => {
-                // The creation should not succeed
+                .then(() => {
+                    // The creation should not succeed
+                    expect.fail();
+                })
+                .catch((error: HTTPResponse) => {
+                    const body: any = JSON.parse(error.body);
+                    expect(body.error).to.equal(`Activity already ended`);
+                });
+        });
+    });
+
+});
+
+describe("getActivities", () => {
```

Comment:
Can you please split the unit tests in multiple files?

Reply:
done